images/win/scripts/Installers/Install-Julia.ps1

```diff
@@ -0,0 +1,6 @@
+################################################################################
+##  File:  Install-Julia.ps1
+##  Desc:  Install Julia
+################################################################################
+
+choco install julia -y
```

Comment:
Julia by default uses `%USERPROFILE%\AppData\Local` for installation. %USERPROFILE% will be deleted after image generation will be finished.
We should choose another folder,  as you mentioned previously: `C:\Julia`.
`choco install julia --ia "/D=C:\Julia"`

Reply:
Fixed in: https://github.com/actions/virtual-environments/pull/386/commits/036ff7c43c3c2e54050d040d3577b479a899b5bc